en/_posts/2016-06-23-linux-and-skylake.md

```diff
@@ -0,0 +1,160 @@
+
+In the installation assistant: if you install a recent enough distibution (like for example Ubuntu 16.04) and make sure you install the ditribution kernel (see screenshot below for this: tick the case 'Install original kernel' in graphic assistant).
+
+![install](/kb/images/2016-06-23-linux-and-skylake/distrib_kernel.png)
+
+If you do not want to follow such a distribution constraint, or did not tick the case during the installation process, no worries, you can still install a kernel meeting the requirements afterward. Besides, your needs may force you to install a kernel compiled with different options than the installed one anyway (for example if you are interested into gaming, you might prefer a lowlatency kernel configuration, instead of the generic one).
+
+We describe some methods about how you can upgrade your kernel if the installed one is too old, from easy ones to advanced ones
+
+
+## Method 1: packaged kernels
+
+Easy and lucky way, the package manager of your distribution (apt for the Debian family, yum/dnf for the RedHat one) can find a repository in your list with compiled kernels embedded into packages.
+
+
+### Example
+
+Ubuntu 14.04, comes by default with an old 3.13 kernel but, if you use the default ubuntu apt repositories (check your /etc/apt/sources/list if not) you can just run:
+
+```
+# apt-get update 
+# mv /etc/grub.d/06_OVHkernel ~/06_OVHkernel.bak 
+# apt-cache search xenial | grep linux-image
+linux-image-extra-virtual-lts-xenial - Transitional package.
+linux-image-generic-lts-xenial - Generic Linux kernel image
+linux-image-lowlatency-lts-xenial - lowlatency Linux kernel image
+linux-image-virtual-lts-xenial - This package will always depend on the latest minimal generic kernel image.
+# apt-get install linux-image-generic-lts-xenial
+[...]
+# shutdown -r now
+```
+
+Then once rebooted verify that you booted on your new kernel and not the previous one. You can check your kernel version by typing
+
+```
+# uname -r
+```
+
+If not check your bootloader file (most likely grub: /boot/grub/grub.cfg) and make sure the default selected menuentry is the good one
+
+
+## Method 2: use a made in ovh bzImage
+
+You can try out our kernels and see if they suit your needs.
+
+[Go to our bzImages repository](ftp://ftp.ovh.net/made-in-ovh/bzImage/)
+
+
+For example you may be interested in
+
+[bzImage-4.5.2-mod-std-ipv6-64](ftp://ftp.ovh.net/made-in-ovh/bzImage/4.5.2/bzImage-4.5.2-mod-std-ipv6-64)
+
+Or (similar to the low latency one showed in method 1, except it has got the drawback of not allowing dynamic module load):
+
+[bzImage-4.5.2-xxxx-std-ipv6-64-hz1000](ftp://ftp.ovh.net/made-in-ovh/bzImage/4.5.2/bzImage-4.5.2-xxxx-std-ipv6-64-hz1000)
+
+
+Please follow the already [existing guide](http://help.ovh.com/KernelInstall) for this, section 'Procedure to up date the kernel on your hard disk'
```

Comment:
Not a bad idea: i need to make sure of the availability of either the kvm or the ipmi sol for this kind of profile first. Otherwise i can at least mention the rescue to revert